packages/neos-ui/src/Containers/Drawer/UserDropDown/UserImage.js

```diff
@@ -0,0 +1,26 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+
+import {neos} from '@neos-project/neos-ui-decorators';
+
+import style from './style.module.css';
+
+@connect(state => ({
+    userFirstName: state?.user?.name?.firstName,
+    userLastName: state?.user?.name?.lastName
+}))
+@neos()
+export default class UserImage extends PureComponent {
+    static propTypes = {
+        userFirstName: PropTypes.string.isRequired,
+        userLastName: PropTypes.string.isRequired
```

Comment:
```suggestion
        userFirstName: PropTypes.string,
        userLastName: PropTypes.string
```

In L21 both props are treated as optional (which makes sense).

Reply:
Applied :)